app/views/comments/_edit.html.erb

```diff
@@ -0,0 +1,8 @@
+
+<div id="update_form">
+<%= form_for [product, comment], remote: :true, method: :PUT do |f| %>
+		<%= hidden_field_tag :authenticity_token, form_authenticity_token %>
```

Comment:
why do we need to send CSRF Token? rails already send this for you in every request